Lesson4/src/com/home/model/MilitaryUnit.java

```diff
@@ -50,4 +51,13 @@ public String getName() {
         ALREADY_IN,
         SUCCESS;
     }
+
+    public void sortListPerson() {
+        String[] tmpArrPerson = new String[this.personSet.size()];
+        for (int i = 0; i < tmpArrPerson.length; i++) {
+            tmpArrPerson[i] = i + ". " + this.personSet.get(i).getSurname() + " " + this.personSet.get(i).getName();
+        }
+        Arrays.sort(tmpArrPerson);
```

Comment:
this will sort people wrong because the element of array starts not from surname bot from the index 1. www, 2.aaa ...

Reply:
this is still not fixed

Reply:
fix

Reply:
now it is sorted correctly, but when you are printing the list you miss index or recruit

I want to see the printed list with an index like

1. Aaa Aaa
2. Aab Aaa
3. Bbb Ccc
and so on. So surnames should be sorted alphabetically, and every line should start from index